neo/Prompt/Utils.py

```diff
@@ -148,6 +148,15 @@ def get_to_addr(params):
     return params, to_addr
 
 
+def get_return_type_from_args(params):
+    return_type = None
+    for item in params:
+        if '--return-type' in item:
+            params.remove(item)
+            return_type = item.replace('--return-type=', '')
```

Comment:
I recommend adding some logic here to ensure only acceptable ParameterTypes are allowed.

Reply:
I asked myself when adding this option, does it add more value to add some sort of input sanity check rather than just educating the user through hitting an exception when they enter a bad parameter type? This is for advanced users after all. Personally I like the feedback loop approach, it doesn't really have any negative impact and it cuts down on the amount of code that needs to be maintained.

Reply:
While true that it reduces code maintenance, we put in a lot of effort to get rid of exception throwing due to user input errors when we refactored the cli. I personally don't think that an advanced option or tool justifies not handling exceptions. In this case it's also as simple as doing

```python
try:
   [ContractParameter.AsParameterType(ContractParameterType.FromString(return_type), item).ToJson() for item in results]
except ValueError:
    # inform user
```